src/constants/tokenSafety.tsx

```diff
@@ -56,8 +56,7 @@ export function getWarningCopy(warning: Warning | null, plural = false) {
 export type Warning = {
   level: WARNING_LEVEL
   message: JSX.Element
-  /* canProceed determines whether triangle/slash alert icon is used, and
-    whether this token is supported/able to be traded */
+  /** Determines whether triangle/slash alert icon is used, and whether this token is supported/able to be traded. */
```

Comment:
Nit: this one could probably be swapped to //

Reply:
This is a doc-comment - using `/**` ensures an IDE will surface this as documentation for the property.